homeassistant/components/renson/services.yaml

```diff
@@ -0,0 +1,143 @@
+set_timer_level:
+  fields:
+    timer_level:
+      required: true
+      advanced: false
+      default: "level1"
+      selector:
+        select:
+          translation_key: "fan_setting"
+          options:
+            - "level1"
+            - "level2"
+            - "level3"
+            - "level4"
+            - "holiday"
+            - "breeze"
+    time:
+      required: true
+      advanced: false
+      default: 0
+      selector:
+        number:
+          min: 0
+          max: 1440
+          step: 10
+          unit_of_measurement: "min"
+          mode: slider
+
+set_breeze:
+  fields:
+    breeze_level:
+      required: false
+      advanced: false
+      default: "level3"
+      selector:
+        select:
+          translation_key: "fan_setting"
+          options:
+            - "level1"
+            - "level2"
+            - "level3"
+            - "level4"
+    temperature:
+      required: false
+      advanced: false
+      default: 18
+      selector:
+        number:
+          min: 15
+          max: 35
+          step: 1
+          unit_of_measurement: "°C"
+          mode: slider
+    activate:
+      required: true
+      advanced: false
+      default: false
+      selector:
+        boolean:
+
+set_day_night_time:
+  fields:
+    day:
+      required: true
+      advanced: false
+      default: "7:00:00"
+      example: "7:00:00"
+      selector:
+        time:
+    night:
+      required: true
+      advanced: false
+      default: "22:00:00"
+      example: "22:00:00"
+      selector:
+        time:
+
+set_pollution_settings:
+  fields:
+    day_pollution_level:
+      required: false
+      advanced: false
+      default: "level3"
+      selector:
+        select:
+          translation_key: "fan_setting"
```

Comment:
I would suggest renaming the translation key to something valid for all fields like `level_settings`.
It is a little bit confusing when the field name is `day_pollution_level` and the translation key `fan_setting`, it looks more like a copypate error.